API-LocateContractor/Controllers/BusinessController.cs

```diff
@@ -92,7 +92,10 @@ public ObjectResult Delete(string userEmail)
                     _businessContext.SaveChanges();
                     return StatusCode(StatusCodes.Status200OK, business);
                 }
-                return StatusCode(StatusCodes.Status304NotModified, business);
+                else
+                {
+                    return StatusCode(StatusCodes.Status304NotModified, business);
```

Comment:
Don't want to use a 304 here. that's for cached files that can be used when the underlying resource has not been modified